plugins/chat/app/jobs/regular/chat/update_thread_reply_count.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Jobs
+  module Chat
+    class UpdateThreadReplyCount < Jobs::Base
+      def execute(args = {})
+        thread = ::Chat::Thread.find_by(id: args[:thread_id])
+        return if thread.blank?
+        return if thread.replies_count_cache_recently_updated?
```

Comment:
It's a little bit annoying to fail silently here IMO

Reply:
What would you do instead? Any logging would be debug-level, warn would make too much noise.

Reply:
It depends what are the reasons this would happen. Is this normal usage of the service or unexpected error? If unexpected than it should be loud.

Reply:
It's normal usage of the job, if the DB cache for `replies_count` was recently updated then we don't do it again, this replicates the debounced/unique sidekiq job essentially.